src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import { getArticles } from "../api"
+
+const Articles = () => {
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    
+    useEffect(() => {
+        console.log("useEffect triggered")
```

Comment:
Remember to take out console logs before making a PR!

Reply:
removed for the next commit, thanks ✅